package.json

```diff
@@ -110,11 +110,13 @@
     "@rails/ujs": "6.1.4",
     "@rails/webpacker": "5.4.3",
     "@reach/combobox": "^0.16.3",
+    "@storybook/addon-controls": "^6.3.12",
     "ahoy.js": "^0.4.0",
     "autoprefixer": "^10.4.0",
     "babel-preset-preact": "^2.0.0",
     "canvas": "^2.8.0",
     "chart.js": "^3.6.0",
+    "classnames": "^2.3.1",
```

Comment:
it's such a small package I agree it's useful to have it instead of long custom functions for the same